hugegraph-core/src/main/java/org/apache/hugegraph/type/Nameable.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.hugegraph.type;
 
 /**
+ * TODO: lack license header & LICENSE reference
```

Comment:
https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/core/Namifiable.java

ditto, should I Change the license to janusgraph license?should I  modify the license?